electrum_nmc/electrum/names.py

```diff
@@ -1007,6 +1039,9 @@ def get_domain_records_txt(domain, value):
     if domain.startswith("_tor."):
         domain = domain[len("_tor."):]
         return get_domain_records_address_tor(domain, value)
+    elif domain.startswith("_ipfs."):
```

Comment:
Prefix should be `_dnslink` per spec at https://docs.ipfs.tech/concepts/dnslink/ unless I'm misunderstanding something?